core/shared/src/test/scala-3/me/shadaj/scalapy/py/SpecialSyntaxTest.scala

```diff
@@ -0,0 +1,53 @@
+package me.shadaj.scalapy.py
+
+import me.shadaj.scalapy.interpreter
+
+import org.scalatest.funsuite.AnyFunSuite
+
+class SpecialSyntaxTest extends AnyFunSuite {
```

Comment:
Should not need a separate Scala 3 version for this test

Reply:
So here should we share this test between two versions? If yes, then we have problems with it, because SpecialSyntaxTest in scala 2 version contains `Can access and update the list elements using brackets` test which is related to bracket syntax (`PyBracketAccess`) and bracketSyntax is not supported in scala 3 at the moment. I can try separate it when I will implement handling annotations in Scala 3, but this separation can be like in the next update

Reply:
I think we should be able to handle this by having two different implementations of a `SpecialSyntaxTestScala2` trait. In Scala 2 it can have `PyBracketAccess` tests and in Scala 3 it would be empty.